aospy/test/test_basic_calc.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python
+"""
+Basic test of the Calc module on 2D data.
+"""
+import unittest
+import shutil
+import os
+from os.path import isfile
+
+from aospy.calc import Calc, CalcInterface
+from local_test_objs.projects import example
+from local_test_objs.models import im
+from local_test_objs.runs import im_example
+from local_test_objs.variables import condensation_rain, precip
+from local_test_objs.regions import globe, sahel
+
+
+class TestBasicCalc(unittest.TestCase):
+    def setUp(self):
+        print os.getcwd()
```

Comment:
Did you want to keep this in?  If so, include `from __future__ import print_function` and then use `print(os.getcwd())` for Python 3 compatibility.